### PR TITLE
Add link to asset page that links to asset position on map search page

### PR DESCRIPTION
**Pull request by Arth101**
Toggl: `Fra asset-kortet til det store kort`

When the link is used from the asset page, we set the asset param and it shows the asset-popup on the map.

### Screenshot:
Screenshot from 2023-04-21 17-26-05

### CODE REVIEW

**Comment by hypesystem** [comment]
Design-wise I don't like the link. Maybe some shorter text would help. Could it be a button next to "Rediger" that says "Vis på stort kort"?

I also think the extra text results in the box on the map being off-center height-wise - would be solved by the above.

File: assets-pipeline/app/scripts-browserify/map-controller.js [hunk 1]
```diff
@@ -295,6 +296,11 @@ function MapController (mapElement, searchControllerCallbacks, options) {
 
       // Plot the assets on the map.
       defaultMapHandler.show(assets);
+      if(options.selectedAsset) {
+        defaultMapHandler.showSelecteedAssetFromParam();
```
Comment (hypesystem) [inline assets-pipeline/app/scripts-browserify/map-controller.js:300]: `showSelect{ee->e}dAssetFromParam`

**Review by hypesystem — APPROVED** [review]
Nice implementation, good code, a single typo.

My design review is the most significant, but this is OK to merge pending a pass on the design.

**Comment by Arth101** [comment]
> Design-wise I don't like the link. Maybe some shorter text would help. Could it be a button next to "Rediger" that says "Vis på stort kort"?

I asked Signe for the link text, so that's why its long. I do like it being a link instead of a button, since it indicates that the action will move you away from the current page. I don't really have strong opinions about this though, just went with my gut.

> I also think the extra text results in the box on the map being off-center height-wise - would be solved by the above.

The overlay-box on the map was never quite centered (only if the location data is sparse since the box has `min-height: 300px`).
See screenshot without the link:
Screenshot from 2023-04-24 12-04-00

I will try to move the link/button around and see what works best :).

**Comment by Arth101** [comment]
Updated with new link position and look: 
image

**Pull request merged**